packages/components/src/DateTimePickers/InputDatePicker/InputDatePicker.scss

```diff
@@ -1,15 +1,16 @@
-$tc-datetimepicker-width: 31rem !default;
-$tc-datetimepicker-background: $white !default;
-$tc-datetimepicker-box-shadow: $shadow-default !default;
-$tc-datetimepicker-z-index: $zindex-dropdown !default;
+$tc-datepicker-width: 31rem !default;
+$tc-datepicker-background: $white !default;
+$tc-datepicker-box-shadow: $shadow-default !default;
+$tc-datepicker-z-index: $zindex-dropdown !default;
 
 .popper {
-	width: $tc-datetimepicker-width;
-	background: $tc-datetimepicker-background;
-	box-shadow: $tc-datetimepicker-box-shadow;
-	z-index: $tc-datetimepicker-z-index;
+	width: $tc-datepicker-width;
+	background: $tc-datepicker-background;
+	box-shadow: $tc-datepicker-box-shadow;
+	z-index: $tc-datepicker-z-index;
 }
 .date-picker {
+	width: fit-content;
```

Comment:
This is to make wrapper(FocusManager) of date input shrink, so click outside of input won't open picker.

Reply:
Sorry I don't understand, can you rephrase ?

Reply:
https://www.loom.com/share/f8a2ef679b234e409a15bb756f2af4d1
I made a screen recording, if I don't set width, click the whole line will open the picker. I want to only open picker when click on input field.